docs/features/kubernetes/configuration.md

```diff
@@ -17,6 +17,9 @@ The following is a full example entry in `app-config.yaml`:
 
 ```yaml
 kubernetes:
+  frontend:
+    podDelete:
+      enabled: true
```

Comment:
@mclarke47 it might be nicer to skip the `frontend` level here, so just `kubernetes.podDelete.enabled`. Or it could even be flipped around to something like `kubernetes.enabledActions.podDelete`. I don't mind too much either way though, there's not really an established pattern here so far.

Reply:
the `frontend` bit was raised by him a while ago.
<img width="822" alt="image" src="https://github.com/user-attachments/assets/122df281-c3d9-45ce-8674-f9e27e6500fd">